src/main/resources/edu/cornell/kfs/coa/cu-ojb-coa.xml

```diff
@@ -1978,6 +2004,10 @@
     	<collection-descriptor name="subAccountGlobalDetails" element-class-ref="edu.cornell.kfs.coa.businessobject.SubAccountGlobalDetail" collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" auto-retrieve="true" auto-update="object" auto-delete="none" proxy="true">
     		<inverse-foreignkey field-ref="documentNumber" />
   		</collection-descriptor>
+  		
+  		<collection-descriptor name="subAccountGlobalNewAccountDetails" element-class-ref="edu.cornell.kfs.coa.businessobject.SubAccountGlobalNewAccountDetail" collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" auto-retrieve="true" auto-update="object" auto-delete="object">
+    		<inverse-foreignkey field-ref="documentNumber" />
```

Comment:
I'm not sure if it's needed here, given that there's no ordering on the subAccountGlobalDetails list above it. Let me know if you think it would still make sense to add ordering here.

Reply:
might as well stay consistent with existing code .... if no one is complaining the order being different must not matter.